src/test/java/teammates/test/cases/browsertests/BaseUiTestCase.java

```diff
@@ -62,6 +49,18 @@ protected void releaseBrowser() {
     }
 
     /**
+     * Reminder to disable GodMode and re-run the test(s).
+     */
+    @AfterSuite
+    public static void remindUserToDisableGodModeIfRequired() {
+        if (TestProperties.IS_GODMODE_ENABLED) {
```

Comment:
Not so important, but perhaps `TestProperties.IS_GODMODE_ENABLED && !TestProperties.isCiEnvironment()`.